### PR TITLE
Add script to update site setup

### PR DESCRIPTION
**Pull request by mkangia**
This could be run when we set up new environments

### CODE REVIEW

File: corehq/apps/hqadmin/management/commands/update_site_setup.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        if not options['skip_checks']:
+            if settings.SITE_ID != 1:
+                raise CommandError("SITE ID under settings expected to have value 1 since only one object is "
+                                   "expected")
+            sites_count = Site.objects.count()
+            if sites_count != 1:
+                raise CommandError("Expected to have only one object added by Site during setup but currently "
+                                   "its %s " % Site.objects.count())
+            site_object = Site.objects.first()
+            if site_object.name != "example.com" and site_object.domain != "example.com":
```
Comment (esoergel) [inline corehq/apps/hqadmin/management/commands/update_site_setup.py]: You could also make this idempotent - check if `name` and `domain` are already right, and then just return.

**Review by esoergel — APPROVED** [review]
Nice! Know what it'd look like to add an ansible task to call this on setup?

File: corehq/apps/hqadmin/management/commands/update_site_setup.py (hunk 1 above)
Comment (mkangia) [reply corehq/apps/hqadmin/management/commands/update_site_setup.py]: Added a check for it as well.

**Comment by mkangia** [comment]
> Nice! Know what it'd look like to add an ansible task to call this on setup?

Not sure. Would need to check.

File: corehq/apps/hqadmin/management/commands/update_site_setup.py [hunk 2]
```diff
@@ -36,6 +36,10 @@ def handle(self, site_address, *args, **options):
                     """.format(name=site_object.name, domain=site_object.domain)
                 )
 
+            if site_object.name == site_object.domain == site_address:
+                print('Site object with the expect name and domain already set.')
+                print('No update made.')
```
Comment (esoergel) [inline corehq/apps/hqadmin/management/commands/update_site_setup.py]: This should `return`

Comment (mkangia) [reply corehq/apps/hqadmin/management/commands/update_site_setup.py]: oops. Adding.

**Review by esoergel — APPROVED** [review]

**Pull request merged**